### PR TITLE
Bump decidim from v0.28.0 to v0.28.1

### PR DESCRIPTION
**Pull request by andreslucena**
As the title says, there's a new version of Decidim so we need to update

### CODE REVIEW

**Comment by andreslucena** [comment]
To finish the update process, once this is deployed in the production database we'll need to run 

`bin/rails decidim:upgrade:fix_orphan_categorizations` 

See more information at https://github.com/decidim/decidim/releases/tag/v0.28.1

**Review by alecslupu — APPROVED** [review]
Lgtm.

@andreslucena  please merge when you need.

File: package.json
Comment (fblupi) [inline package.json:1]: As we are not using the `file:packages/dir` anymore here, we could remove the `packages` directory

**Review by fblupi — CHANGES_REQUESTED** [review]

File: package.json
Comment (andreslucena) [reply package.json:1]: Done at 4efbd08c2032705e2b3587a3b57e1f141480a5cb

**Review by fblupi — COMMENTED** [review]
@andreslucena I receive this error while compiling the assets.

```
(58:6) /Users/fbo/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/decidim-meetings-0.28.1/app/packs/stylesheets/decidim/meetings/_item.scss The `min-w-24` class does not exist. If `min-w-24` is a custom class, make sure it is defined within a `@layer` directive.

  56 | 
  57 |     &__lg {
> 58 |       @apply w-fit justify-center [&>*]:px-2 min-w-24 h-[8.5rem];
     |      ^
  59 |     }
  60 | 
```

I tried deploying it in Heroku thinking it was something on my local, but it's failing with the same error too.

I don't get why it's not failing in the CI then.

**Comment by andreslucena** [comment]
> (58:6) /Users/fbo/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/decidim-meetings-0.28.1/app/packs/stylesheets/decidim/meetings/_item.scss The `min-w-24` class does not exist. If `min-w-24` is a custom class, make sure it is defined within a `@layer` directive.

I can reproduce this same error locally with 

`SECRET_KEY_BASE=potato RAILS_ENV=production bin/rails assets:precompile`

I think it's related to the tailwindcss version that we're using and the backport that introduced this change in v0.28 (#12787). This is using a directive that isn't available in the version of tailwind that we're using in v0.28.

I made a quick fix in bfb306cf7cadf6ad9b8602c16b327d404bd908e7, just to check out if this is indeed the problem, so we can also introduce that change in `release/0.28-stable` (and do a v0.28.2 with this fix). 

This is something that @microstudi mentioned a couple of weeks ago, but I could only reproduce in this app with this change. Locally with a new v0.28.1 app I could not reproduce this bug, as I see that it generates the app with tailwind 3.4.4. 

I think the actual issue is that we're using the caret character (`^`) and this applies to minor and patch version, and we should use the tilde character (`~`). With that, I understand that we would have detected the problem in CI (as we should have done).

**Review by fblupi — APPROVED** [review]
I have successfully executed it locally after applying the latest change.

Seems there's an error in the CI but, as it's not related with this and the tests run successfully locally, I approve it.

**Pull request merged**